new-integration-tests/src/test/java/oracle/weblogic/kubernetes/Mark.java

```diff
@@ -0,0 +1,70 @@
+// Copyright (c) 2020, Oracle Corporation and/or its affiliates.
+// Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl.
+
+package oracle.weblogic.kubernetes;
+
+import io.kubernetes.client.openapi.ApiException;
+import oracle.weblogic.kubernetes.annotations.IntegrationTest;
+import oracle.weblogic.kubernetes.extensions.LoggedTest;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import static oracle.weblogic.kubernetes.actions.TestActions.createSecret;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatCode;
+import static org.assertj.core.api.Assertions.catchThrowable;
+
+@DisplayName("Simple validation of basic domain functions")
+@IntegrationTest
+class Mark implements LoggedTest {
+
+  @Test
+  @DisplayName("Test something")
+  public void testSomeStuff() {
+    int x = 1 + 5;
+    logger.info("log message");
+
+    assertThat(true).as("Test that true is true")
+        .withFailMessage("OMG! True was not true")
+        .isEqualTo(true);
+  }
+
+  @Test
+  @DisplayName("Test something else")
+  public void testThereWasNoException() {
+    // this test will fail because it does throw an exception!
+
+    assertThatCode(() ->
+        createSecret("bob", // name
+            "weblogic", // username
+            "welcome1", // password
+            "default")) //namespace
+        .as("Test that createSecret does not throw an exception")
+        .withFailMessage("OMG! createSecret() threw an unexpected exception")
+        .doesNotThrowAnyException();
+  }
+
+  @Test
+  @DisplayName("Test catching an exception")
+  public void testCatchingAnException() {
```

Comment:
Looks like Junit 5's default display name generator can be replaced so as to make method/class names look pretty in the logging.  [Here's a camel case one](https://www.baeldung.com/junit-custom-display-name-generator) that parses the method name and would generate something that closely matches the custom display name you had to type in by hand here.

Reply:
(That said I'd personally prefer just to see the method name and a timestamp.)

Reply:
That's a fair point @tbarnes-us -- the Jenkins plugins that I use ignore the display name anyway:

![Screenshot from 2020-04-03 23-22-00](https://user-images.githubusercontent.com/6345393/78417644-1efb0180-7602-11ea-88cb-e15a1ff2ba2b.png)


Though the output does have both: 
```
[INFO] Running oracle.weblogic.kubernetes.ItMark
<04-04-2020 03:20:55> <INFO> <oracle.weblogic.kubernetes.extensions.LoggedTest beforeEachTest> <About to execute [Test catching an exception] in oracle.weblogic.kubernetes.ItMark.testCatchingAnException()>
<04-04-2020 03:20:55> <INFO> <oracle.weblogic.kubernetes.extensions.Timing afterTestExecution> <Method [testCatchingAnException] took 223 ms.>
<04-04-2020 03:20:55> <INFO> <oracle.weblogic.kubernetes.extensions.LoggedTest afterEachTest> <Finished executing [Test catching an exception] in oracle.weblogic.kubernetes.ItMark.testCatchingAnException()>
<04-04-2020 03:20:55> <INFO> <oracle.weblogic.kubernetes.extensions.LoggedTest beforeEachTest> <About to execute [Test something] in oracle.weblogic.kubernetes.ItMark.testSomeStuff()>
<04-04-2020 03:20:55> <INFO> <oracle.weblogic.kubernetes.ItMark testSomeStuff> <log message>
<04-04-2020 03:20:55> <INFO> <oracle.weblogic.kubernetes.extensions.Timing afterTestExecution> <Method [testSomeStuff] took 2 ms.>
<04-04-2020 03:20:55> <INFO> <oracle.weblogic.kubernetes.extensions.LoggedTest afterEachTest> <Finished executing [Test something] in oracle.weblogic.kubernetes.ItMark.testSomeStuff()>
<04-04-2020 03:20:55> <INFO> <oracle.weblogic.kubernetes.extensions.LoggedTest beforeEachTest> <About to execute [Test something else] in oracle.weblogic.kubernetes.ItMark.testThereWasNoException()>
<04-04-2020 03:20:55> <INFO> <oracle.weblogic.kubernetes.extensions.Timing afterTestExecution> <Method [testThereWasNoException] took 170 ms.>
<04-04-2020 03:20:55> <INFO> <oracle.weblogic.kubernetes.extensions.LoggedTest afterEachTest> <Finished executing [Test something else] in oracle.weblogic.kubernetes.ItMark.testThereWasNoException()>
[ERROR] Tests run: 3, Failures: 2, Errors: 0, Skipped: 0, Time elapsed: 0.417 s <<< FAILURE! - in oracle.weblogic.kubernetes.ItMark
[ERROR] oracle.weblogic.kubernetes.ItMark.testCatchingAnException  Time elapsed: 0.226 s  <<< FAILURE!
java.lang.AssertionError: [Test that createSecret does not throw an APIException] OMG! createSecret() threw an APIException
	at oracle.weblogic.kubernetes.ItMark.testCatchingAnException(ItMark.java:65)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
     ...
```

Honestly, I just put them in there in the first place because I was trying out various JUnit features, and then they got copied.  

In another project, where we use different tools, those types of names are more useful.. I dropped an image in slack to show you what I mean,

I am ok to drop them here, or leave them in, as the team feels best.  I don't think they hurt any - you get both the class/method and the display name in the output.

Reply:
Fine by me. It's nice to see the default is generally to show the timestamp/class-name/method-name _plus_ the display name, if any...

A couple of side nits:

1.  I find it a little annoying that the exception reported by [ERROR] is not also included in-line in the logging exactly when it happens, and the exception that is reported doesn't include a timestamp.  This makes it more work to figure out exactly where and when a failure occurred.

2. It'd be nice if the 'took 2 ms' log message included the full path.  
  - Right now it's:
     - <04-04-2020 03:20:55> <INFO> <oracle.weblogic.kubernetes.extensions.Timing afterTestExecution> <Method [testSomeStuff] took 2 ms.>
  - but would be nicer if it was:
     - <04-04-2020 03:20:55> <INFO> <oracle.weblogic.kubernetes.extensions.Timing afterTestExecution> <Method [oracle.weblogic.kubernetes.ItMark.testSomeStuff] took 2 ms.>
  - or even better, reduce the logging verbosity and append the 'took 2ms' at the end of the 'Finished executing' log message instead of having an entire separate log message for 'took'.